java-checks/src/main/java/org/sonar/java/checks/StaticMethodCheck.java

```diff
@@ -67,7 +80,7 @@ public void visitClass(ClassTree tree) {
 
   @Override
   public void visitMethod(MethodTree tree) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "visitMethod" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)